pylibdmtx/pylibdmtx.py

```diff
@@ -37,6 +37,7 @@
 
 # A rectangle
 Rect = namedtuple('Rect', 'left top width height')
+Rect_vertices = namedtuple('Rect', 'P0 P1 P2 P3')
```

Comment:
This gives a mypy error: 
```
First argument to namedtuple() should be "Rect_vertices", not "Rect"  [name-match]mypy(error)
```
But I'm not too familiar with namedtuple's, so I don't know the implications of changing it.

Reply:
Thanks for checking it out. On Python3,8 I didn't have this error. I am not too familiar with namedtuple either, but it shouldn't be a problem to just change it to
```suggestion
Rect_vertices = namedtuple('Rect_vertices', 'P0 P1 P2 P3')
```
if it works for you

Reply:
This is all a bit beyond me, but here's a post on stack overflow regarding the relevance of typenames for namedtuples:
https://stackoverflow.com/questions/22762236/relevance-of-typename-in-namedtuple

And here's a site that says to support pickling, typenames of named tuples should match the variable:
https://symonk-pydocs.readthedocs.io/en/latest/namedtuples.html#:~:text=Namedtuple%3A%20typename%20%C2%B6%20typename%20is%20the%20new%20tuple,of%20the%20variable%20assigned%20to%20the%20tuple%20subclass.

excerpt: typename is the new tuple subclass name. In order for pickling to be natively supported the typename should match the name of the variable assigned to the tuple subclass.

Reply:
Just tested the suggested change, and this is the output:

Decoded(data=b'Stegosaurus', rect=Rect_vertices(P0=(189, 92), P1=(189, 187), P2=(285, 92), P3=(285, 187)))
Stegosaurus
Decoded(data=b'Plesiosaurus', rect=Rect_vertices(P0=(333, 89), P1=(333, 184), P2=(429, 89), P3=(429, 184)))
Plesiosaurus

So, doesn't seem to be any issues making the change.